lib/generators/avo/templates/locales/pagy/ro.yml

```diff
@@ -0,0 +1,17 @@
+# :one_other pluralization (see https://github.com/ddnexus/pagy/blob/master/lib/locales/utils/p11n.rb)
+
+ro:
+  pagy:
+    item_name:
+      one: "un record"
+      other: "recorduri"
+    nav:
+      prev: "&lsaquo;&nbsp;Înapoi"
+      next: "Înainte&nbsp;&rsaquo;"
+      gap: "&hellip;"
```

Comment:
I don't see these working in the UI

![CleanShot 2022-11-26 at 15 37 42](https://user-images.githubusercontent.com/1334409/204091695-74409b86-3fa2-4575-ad9c-b5444d72eb5e.jpg)

Reply:
Did you restart the server after changing to this branch?